flottform/server-standalone/src/routes/flottform/turn-credentials/+server.ts

```diff
@@ -0,0 +1,68 @@
+import { type RequestHandler, json, text } from '@sveltejs/kit';
+import { env } from '$env/dynamic/private';
+import { getUseTurnServer } from '../turn-control/global';
+
+export const GET: RequestHandler = async () => {
+	const useTurnServer = getUseTurnServer();
+	const iceServersEnv = env.ICE_SERVERS_CONFIGURATION;
+
+	if (!iceServersEnv) {
+		return json(
+			{ success: false, message: 'No ICE server configuration found in the environment' },
+			{
+				status: 500,
+				headers: {
+					'Access-Control-Allow-Origin': '*',
+					'Access-Control-Allow-Methods': 'GET',
+					'Access-Control-Allow-Headers': '*'
+				}
+			}
+		);
+	}
```

Comment:
Should we return the default Google iceServer as we did before in case of someone not providing anything here?

Reply:
The If-statement here is to remind us to set the environment variable. 

Even if the GET request doesn't return anything (which is the case here), we initialize the value of rtcConfiguration inside `FlottformChannelHost` and `FlottformChannelClient` to `DEFAULT_WEBRTC_CONFIG` which contains the default Google STUN server. Which means we won't have any issues